spec/std/file_spec.cr

```diff
@@ -329,7 +337,11 @@ describe "File" do
       begin
         File.write(path, "")
         File.chmod(path, 0o775)
-        File.info(path).permissions.should eq(File::Permissions.new(0o775))
+        {% if flag?(:win32) %}
+          File.info(path).permissions.should eq(File::Permissions.new(0o666))
```

Comment:
Windows doesn't have the full permissions model, so i'm not sure how to clean up these specs.